domains/kunsh13.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "kunsh13.is-a.dev",
+  "repo": "https://github.com/kunsh13/iptv",
+  "owner": {
+    "username": "kunsh13",
+    "email": "kunshmaurya@gmail.com"
+  },
+  "record": {
+    "CNAME": "kunsh13.github.io/iptv"
```

```suggestion
    "CNAME": "kunsh13.github.io"
```
CNAMEs cannot contain slashes `/`.

---

But I want a repo of mine to be hosted, how can I do that?